etl/steps/data/garden/who/2023-04-03/flu_elderly.py

```diff
@@ -0,0 +1,54 @@
+"""Load a meadow dataset and create a garden dataset."""
```

Comment:
Maybe a little more context for these steps can help (I would like `walkthrough` did this anyway)